data/json/professions.json

```diff
@@ -397,19 +397,23 @@
             }
         ],
         "items": {
-            "both": [
-                "tank_top",
-                "hoodie",
-                "pants_cargo",
-                "socks",
-                "sneakers",
-                "weed",
-                "tobacco",
-                "lighter",
-                "rolling_paper",
-                "switchblade",
-                "picklocks"
-            ],
+            "both": {
```

Comment:
Sorry, next time a tangent like that comes up, I'll give it its own PR then rebase.